### PR TITLE
simplified rotational sensor helper to one-directional Rotation2d support

### PR DESCRIPTION
**Pull request by R7B9**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/swerve/Swerve.java [hunk 1]
```diff
@@ -85,19 +85,19 @@ public class Swerve extends SubsystemBase implements Tuneable {
...
 
-        gyroYawHelper = new RotationalSensorHelper(
-                gyroIO.isConnected.getAsBoolean() ? gyroIO.yawDegreesCW.getAsDouble() : 0);
+        gyroYawHelperCCW = new RotationalSensorHelper(
+                Rotation2d.fromDegrees(gyroIO.isConnected.getAsBoolean() ? -gyroIO.yawDegreesCW.getAsDouble() : 0));
 
         poseEstimator = new SwerveDrivePoseEstimator(
                 swerveKinematics,
-                gyroYawHelper.getRawMeasuredAngleCCW(),
+                gyroYawHelperCCW.getMeasuredAngle(),
```
Comment (RBD120307) [inline src/main/java/frc/robot/subsystems/swerve/Swerve.java:98]: At the end you decided that RAW isn't necessary?

File: src/main/java/frc/robot/subsystems/swerve/Swerve.java [hunk 2]
```diff
@@ -215,12 +215,8 @@ public void setModulesState(SwerveModuleState[] moduleStates, boolean preventJit
...
     }
 
-    private double getYawDegreesCW() {
-        return gyroYawHelper.getAngleDegreesCW();
-    }
-
-    private double getYawDegreesCCW() {
-        return gyroYawHelper.getAngleDegreesCCW();
+    private Rotation2d getYawCCW() {
+        return gyroYawHelperCCW.getAngle();
```
Comment (RBD120307) [inline src/main/java/frc/robot/subsystems/swerve/Swerve.java:219]: Isn't getRotation2d better?

File: src/main/java/frc/robot/subsystems/swerve/Swerve.java (hunk 1 above)
Comment (R7B9) [reply src/main/java/frc/robot/subsystems/swerve/Swerve.java:98]: yes

File: src/main/java/frc/robot/subsystems/swerve/Swerve.java (hunk 2 above)
Comment (R7B9) [reply src/main/java/frc/robot/subsystems/swerve/Swerve.java:219]: I don't think so. Rotation2d is a type, it doesn't tell you what the value of it means.

**Pull request merged**